src/index.js

```diff
@@ -40,5 +47,33 @@ form.addEventListener('submit', (event) => {
 add.addEventListener('click', () => {
   const name = document.getElementById('name');
   const score = document.getElementById('score');
-  if ((name.value !== '') && (score.value !== '')) { addToPage(name.value, score.value); }
+  if ((name.value !== '') && (score.value !== '')) {
+    addToPage(name.value, score.value);
+    const entry = {
+      user: name.value,
+      score: score.value,
+    };
+    postData(url, entry)
+      .then()
+      .catch((error) => {
+        console.error('Error: ', error);
+      });
+  }
```

Comment:
- [ ] Kindly make sure to render the data coming from the API after the request is successful `status: ok`, you can simply fetch back the score in the empty `then` call back and add each score to the list of scores.

Reply:
Thank you for the review, I will like to point out

The then() block is empty because the data rendered from the API at this point is a simple confirmation message.
{
	"result": "Leaderboard score created correctly."
}

fetching the scores at this point will make the fetch of the scores dependent on the submit button in the form; from the requirements, a fetch of scores should only be done when the user clicks the "refresh" button. The code that handles the score fetch is at the bottom of the index.js with an event listener.